lib/policies/serializationPolicy.ts

```diff
@@ -138,8 +138,25 @@ export class SerializationPolicy extends BaseRequestPolicy {
           }
           return Promise.reject(error);
         }
-      } else {
+      } else if (responseSpec) {
+        if (responseSpec.bodyMapper) {
+          let valueToDeserialize: any = response.parsedBody;
+          if (operationSpec.isXML && responseSpec.bodyMapper.type.name === MapperType.Sequence) {
+            valueToDeserialize = typeof valueToDeserialize === "object" ? valueToDeserialize[responseSpec.bodyMapper.xmlElementName!] : [];
+          }
+          try {
+          response.parsedBody = operationSpec.serializer.deserialize(responseSpec.bodyMapper, valueToDeserialize, "operationRes.parsedBody");
```

Comment:
reindent

Reply:
Fixed in https://github.com/Azure/ms-rest-js/pull/149/commits/b36d99cdac1d84aefe19cd003aaca338aae97810.